packages/cfd/src/Containers/mt5-create-password/mt5-create-password.tsx

```diff
@@ -68,21 +69,25 @@ const MT5CreatePassword = ({
                 validateForm,
             }) => (
                 <form onSubmit={handleSubmit}>
-                    <div className='mt5_password'>
-                        <div
-                            className='cfd-password-modal__content dc-modal__container_cfd-password-modal__body'
-                            data-testid='dt_create_password'
-                        >
-                            <div className='cfd-password-modal__create-password-content'>
-                                <Icon icon='IcMt5OnePassword' width='100' height='100' />
-                            </div>
-                            <Text size='xs' align='left' className='cfd-password-modal__create-password-description'>
+                    <div
+                        className='mt5-password-modal__content dc-modal__container_mt5-password-modal__body form'
+                        data-testid='dt_mt5_create_password'
+                    >
+                        <div className='mt5-password-modal__create-password-content'>
+                            <Icon icon='IcMt5OnePassword' width='150' height='150' />
```

Comment:
Should we use Quill icon here or is it fine ? 😬

Reply:
hey @fasihali-deriv  i will clarify once

Reply:
hey @fasihali-deriv we can implement quill icons here 👌